0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -1,7 +1,104 @@
+
+Accounts in ILQP are identified by their ILP Addresses. For a full explanation of ILP Addresses, see [Proposal: ILP Address Mapping](https://github.com/interledger/rfcs/issues/31) and [Fowarding/Delivery Distinction](https://github.com/interledger/rfcs/issues/77).
+
+<!--{# ***TODO: Better reference for ILP addresses***. #}-->
+
+ILP payments can as simple as two transfers in different ledgers joined by a single connector. In other cases, they may be longer chains with multiple connectors across three or more ledgers. The response of an ILQP request only defines one link in the chain.
+
+**Ledgers:**
+
+* The **source ledger** is where the sender and the connector both have accounts.
+* The **destination ledger** is where the receiver and the connector both have accounts.
+
+The sending and receiving ledger are different ledgers. Otherwise, there should be no reason to use ILQP.
+
+**Parties:**
+
+* The **sender** is the party being debited at the start of the overall chain.
+* The **source** is the party being debited in a single link of the chain. In a simple payment, the source is also the sender.
+* The **receiver** is the party being credited at the end of the overall chain.
+* The **destination** receives money from the connector in a single link. In a simple payment, the destination is also the receiver.
+* A **connector** facilitates the payment between the source and destination in a single link. In a multiple-hop payment, there are multiple connectors, each of which is the destination of one link and the source of the next.
+
+## Get Quote (HTTP)
+
+This is the only API method in the HTTP+JSON version of the ILQP.
+
+### Request Format
 
-**TODO**: What's the best way to link this?
+```
+GET /
+```
+
+The request uses the following query parameters:
+
+| Field                         | Type        | Description                    |
+|:------------------------------|:------------|:-------------------------------|
+| `source_address`              | ILP Address | Address of the source's account in the source ledger. |
+| `destination_address`         | ILP Address | Address of the destination's account in the destination ledger. |
+| `source_amount`               | Number      | Fixed amount to debit from the source's account. (Required unless `destination_amount` specified.) |
+| `destination_amount`          | Number      | Fixed amount to credit to the destination's account. (Required unless `source_amount` specified.) |
+| `destination_expiry_duration` | Number      | (Optional) Number of milliseconds before the transfer in the destination ledger expires. |
+| `source_expiry_duration`      | Number      | (Optional) Number of milliseconds before the transfer in the source ledger expires. |
+| `slippage`                    | Number      | (Optional) Custom slippage ***amount? percentage?*** to assume for the exchange between ledgers. Lower slippage means the payment is cheaper but more likely to fail due to market movement. |
```

Comment:
It's a percentage:

> CONNECTOR_SLIPPAGE (default: 0.001 = 0.1%) The ratio for overestimating exchange rates to prevent payment failure if the rate changes.

From https://github.com/interledger/js-ilp-connector/blob/master/README.md#L106